src/cmd/src/bin/greptime.rs

```diff
@@ -29,6 +29,8 @@ struct Command {
     log_dir: Option<String>,
     #[clap(long)]
     log_level: Option<String>,
+    #[clap(long)]
+    tokio_console_addr: Option<String>,
```

Comment:
Consider putting this option under feature gate so that it won't show up in `--help` for our release build.